src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+let li = document.querySelectorAll('li');
+
+function sortList(list) {
+  const sortedList = Array.from(list).sort((a, b) => {
+    const salaryA = parseFloat(a.getAttribute('data-salary'));
+    const salaryB = parseFloat(b.getAttribute('data-salary'));
+
+    return salaryB - salaryA;
+  });
+
+  return sortedList;
```

Comment:
now sorted list not rendered.
find ul and replace content in ul element on sorted list.